TravelPoints-BackEnd/src/main/java/com/example/travelpointsbackend/model/Review.java

```diff
@@ -0,0 +1,33 @@
+package com.example.travelpointsbackend.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import java.io.Serial;
+import java.io.Serializable;
+import java.sql.Timestamp;
+
+@Entity
+@AllArgsConstructor
+@Builder
+@Data
+@Table(name = "reviews")
+public class Review implements Serializable {
+    @Serial
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long reviewId;
+    private Long userId;
+    private Long attractionId;
+    private String reviewText;
+    private Integer rating;
+    private Timestamp createdAt;
```

Comment:
M: see above, also use mappings for relations between tables. (See here: https://stackabuse.com/a-guide-to-jpa-with-hibernate-relationship-mapping/)